app/views/orders/index.erb

```diff
@@ -1 +1,3 @@
-Orders
\ No newline at end of file
+<% @orders.each do |order| %>
+<p><%= order.name %></p>
```

Comment:
this <p> tag should be indented one level deeper as it's a child element of the iteration above